2025-03-02 22:49:31,967 ERROR: Request finalizing failed with an error while handling an error [in /home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py:901]
Traceback (most recent call last):
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 894, in finalize_request
    response = self.process_response(response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 1274, in process_response
    self.session_interface.save_session(self, ctx.session, response)
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask_session/sessions.py", line 164, in save_session
    response.set_cookie(app.config["SESSION_COOKIE_NAME"], session_id,
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/werkzeug/sansio/response.py", line 224, in set_cookie
    dump_cookie(
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/werkzeug/http.py", line 1303, in dump_cookie
    if not _cookie_no_quote_re.fullmatch(value):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: cannot use a string pattern on a bytes-like object

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 894, in finalize_request
    response = self.process_response(response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 1274, in process_response
    self.session_interface.save_session(self, ctx.session, response)
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask_session/sessions.py", line 164, in save_session
    response.set_cookie(app.config["SESSION_COOKIE_NAME"], session_id,
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/werkzeug/sansio/response.py", line 224, in set_cookie
    dump_cookie(
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/werkzeug/http.py", line 1303, in dump_cookie
    if not _cookie_no_quote_re.fullmatch(value):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: cannot use a string pattern on a bytes-like object
2025-03-02 22:50:46,502 INFO: Document System startup [in /home/monk/Github/dope/EduSync/app/config.py:111]
2025-03-02 22:50:46,602 INFO: Document System startup [in /home/monk/Github/dope/EduSync/app/config.py:111]
2025-03-02 22:50:46,742 INFO: Document System startup [in /home/monk/Github/dope/EduSync/app/config.py:111]
2025-03-02 22:50:46,776 INFO: Document System startup [in /home/monk/Github/dope/EduSync/app/config.py:111]
2025-03-02 22:51:12,971 INFO: Document System startup [in /home/monk/Github/dope/EduSync/app/config.py:111]
2025-03-02 22:51:15,653 ERROR: Exception on / [GET] [in /home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py:828]
Traceback (most recent call last):
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 894, in finalize_request
    response = self.process_response(response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 1274, in process_response
    self.session_interface.save_session(self, ctx.session, response)
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask_session/sessions.py", line 164, in save_session
    response.set_cookie(app.config["SESSION_COOKIE_NAME"], session_id,
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/werkzeug/sansio/response.py", line 224, in set_cookie
    dump_cookie(
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/werkzeug/http.py", line 1303, in dump_cookie
    if not _cookie_no_quote_re.fullmatch(value):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: cannot use a string pattern on a bytes-like object
2025-03-02 22:51:15,681 ERROR: Request finalizing failed with an error while handling an error [in /home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py:901]
Traceback (most recent call last):
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 894, in finalize_request
    response = self.process_response(response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 1274, in process_response
    self.session_interface.save_session(self, ctx.session, response)
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask_session/sessions.py", line 164, in save_session
    response.set_cookie(app.config["SESSION_COOKIE_NAME"], session_id,
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/werkzeug/sansio/response.py", line 224, in set_cookie
    dump_cookie(
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/werkzeug/http.py", line 1303, in dump_cookie
    if not _cookie_no_quote_re.fullmatch(value):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: cannot use a string pattern on a bytes-like object

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 894, in finalize_request
    response = self.process_response(response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 1274, in process_response
    self.session_interface.save_session(self, ctx.session, response)
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask_session/sessions.py", line 164, in save_session
    response.set_cookie(app.config["SESSION_COOKIE_NAME"], session_id,
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/werkzeug/sansio/response.py", line 224, in set_cookie
    dump_cookie(
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/werkzeug/http.py", line 1303, in dump_cookie
    if not _cookie_no_quote_re.fullmatch(value):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: cannot use a string pattern on a bytes-like object
2025-03-03 09:52:42,022 INFO: Document System startup [in /home/monk/Github/dope/EduSync/app/config.py:111]
2025-03-03 09:52:45,302 ERROR: Exception on / [GET] [in /home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py:828]
Traceback (most recent call last):
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 894, in finalize_request
    response = self.process_response(response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask/app.py", line 1274, in process_response
    self.session_interface.save_session(self, ctx.session, response)
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/flask_session/sessions.py", line 164, in save_session
    response.set_cookie(app.config["SESSION_COOKIE_NAME"], session_id,
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/werkzeug/sansio/response.py", line 224, in set_cookie
    dump_cookie(
  File "/home/monk/Github/dope/EduSync/venv/lib/python3.12/site-packages/werkzeug/http.py", line 1303, in dump_cookie
    if not _cookie_no_quote_re.fullmatch(value):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: cannot use a string pattern on a bytes-like object
